assets/govuk_frontend_toolkit/stylesheets/_colours.scss

```diff
@@ -103,8 +103,10 @@ $panel-colour: $grey-3;           // Related links panel, page footer etc.
 $canvas-colour: $grey-4;          // Page background
 $highlight-colour: $grey-4;       // Table stripes etc.
 $page-colour: $white;             // The page
-$alpha-colour: $pink;             // Alpha badges and banners
-$beta-colour: $orange;            // Beta badges and banners
+$discovery-colour: $govuk-blue;   // Discovery badges and banners
```

Comment:
It looks like `$discovery-colour` is only used in the documentation site. Do we need it?

Reply:
https://github.com/alphagov/govuk_elements/search?utf8=%E2%9C%93&q=discovery-colour&type=

Reply:
It doesn't affect anything we're doing, to remove this and $live-colour. They'll appear again in AF v3/4 anyhow if they're to be a copy of gov.uk assets.

Reply:
Yes very true! Not having to worry about them now simplifies this PR to a single concern is all.